modules/claims_api/config/schemas/v2/2122a.json

```diff
@@ -221,6 +221,7 @@
       "additionalProperties": false,
       "required": [
         "poaCode",
+        "registrationNumber",
```

Comment:
we could probably remove first and last name from here. That would require adding them back into the form data like what you did for 2122.